<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuVentas</class>
 <widget class="QMainWindow" name="MenuVentas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuVentas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Ventas.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddVenta">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QPushButton{

	background-color: rgb(255, 255, 255) ;
	
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}

QPushButton:hover{

	background-color:  rgb(85, 170, 0);
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_consultVenta">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(255, 255, 255) ;
	
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}

QPushButton:hover{

	background-color: rgb(255, 170, 0);
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
    </property>
    <property name="text">
     <string>Consultar </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ReportVentas">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(255, 255, 255) ;
	
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}

QPushButton:hover{

	background-color:rgb(255, 255, 0);
	font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
	
}</string>
    </property>
    <property name="text">
     <string>Reporte de Ventas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Back">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;Bahnschrift SemiBold&quot;;
back-groud color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ir al men√∫ principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
